both/collections/competition.js

```diff
@@ -100,9 +120,7 @@ Competitions.attachSchema({
         return "missingRegistrationOpenDate";
       }
 
-      if(registrationOpenDate.getTime() < registrationCloseDate.getTime()) {
-        return null;
-      } else {
+      if(registrationOpenDate.getTime() >= registrationCloseDate.getTime()) {
```

Comment:
I didn't know how much you would like `validationObject`, so I didn't change every validation to use it.

But I'll do that if you like the pattern!